dist/prometheus/chihaya.rules

```diff
@@ -0,0 +1,74 @@
+# Chihaya recording rules
```

Comment:
We should definitely have documentation on what rules are (how they are optional) etc...